js/opening-hours.functions.js

```diff
@@ -353,59 +718,76 @@ OpeningHours.prototype.calendarEvents = function (element, settings) {
 /**
  * Handle keyboard input to move to other dates.
  *
- * @param {Event} e
+ * @param {KeyboardEvent} event
  *   The keydown event.
- * @param {HTMLElement} elem
+ * @param {HTMLElement} element
  *   Wrapper element which contains the days.
  */
-OpeningHours.prototype.handleKeyboardInput = function (e, elem) {
-  let keyCode = e.keyCode || e.which;
-  let current = e.target;
-  let currentPosition = +current.getAttribute('aria-posinset');
+OpeningHours.prototype.handleKeyboardInput = function (event, element) {
+  var key = event.key;
```

Comment:
(Not sure if this is cross-browser compatible? See implementation notes on: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Browser_compatibility)